### PR TITLE
Added examples for github:repo:create action and improved test cases

### PR DESCRIPTION
**Pull request by parmar-abhinav**
#### :heavy_check_mark: Checklist

Contribution of publish:azure examples as part of https://github.com/backstage/backstage/issues/18832
- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by parmar-abhinav** [comment]
@Rugvip @acierto Could you please review this PR.

File: plugins/scaffolder-backend-module-github/src/actions/githubRepoCreate.examples.ts [hunk 1]
```diff
@@ -63,4 +63,626 @@ export const examples: TemplateExample[] = [
...
+          input: {
+            repoUrl: 'github.com?repo=repo&owner=owner',
+            repoVisibility: 'internal',
+            requiredCommitSigning: true,
+          },
+        },
+      ],
+    }),
+  },
+  // {
```
Comment (acierto) [inline plugins/scaffolder-backend-module-github/src/actions/githubRepoCreate.examples.ts:679]: Can you please remove the commented code?

**Review by acierto — CHANGES_REQUESTED** [review]

File: plugins/scaffolder-backend-module-github/src/actions/githubRepoCreate.examples.ts (hunk 1 above)
Comment (parmar-abhinav) [reply plugins/scaffolder-backend-module-github/src/actions/githubRepoCreate.examples.ts]: Thank you for reviewing, requested changes are done. Please check

**Comment by parmar-abhinav** [comment]
@acierto Completed the requested changes. Could you please review the PR now

**Comment by parmar-abhinav** [comment]
@benjdlambert The error handling while parsing yaml is suggested in my earlier PR as a part of refactoring code.

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**